### DISCUSSION TITLE
评论功能必须填写用户信息么？可以直接直接评论不

### DISCUSSION THREAD

**Discussion by duguyue8** [halo-dev/halo#4564] [Q&A]
![问题图片](https://pic7.58cdn.com.cn/nowater/webim/big/n_v23958403bcf3e4dfd9e8681bd2bd10c5a.png)

**Comment by JohnNiang** [comment]
可在这里配置评论：

<img width="642" alt="image" src="https://github.com/halo-dev/halo/assets/16865714/cea39573-35ef-4194-bbe1-b653becf0053">

目前，若评论没有登录，则必须填写昵称和电子邮箱（可任意填写）。
